test/trailing-slash.test.ts

```diff
@@ -1,7 +1,27 @@
 // @ts-nocheck
 import { withTrailingSlash, withoutTrailingSlash } from '../src'
 
-describe('withTrailingSlash', () => {
+describe('withTrailingSlash, queryParams: false', () => {
+  const tests = {
+    '': '/',
+    bar: 'bar/',
+    'bar/': 'bar/',
+    'foo?123': 'foo?123/',
+    'foo/?123': 'foo/?123/'
+  }
+
+  for (const input in tests) {
+    test(input, () => {
+      expect(withTrailingSlash(input)).toBe(tests[input])
```

Comment:
Has the behavior changed for the case when boolean argument is not provided?

Old code:
`'foo?123': 'foo?123/',`
New code:
`'foo?123': 'foo?123/',`

I thought that the point of introducing the boolean was not to change the original behavior?

Also, I don't see how a change like `'foo?123': 'foo?123/',` is useful or valid.

Reply:
It was not changed. The default behavior is `'foo?123': 'foo?123/'` ,

I have just pushed changes to `trailing-query` branch which looks different.

Thanks!

Reply:
Very confused now (one PR pushing changes to another PR ;)).

Looking at changes in https://github.com/unjs/ufo/pull/24 it shows that that test was newly added so makes sense I guess.

Still would prefer a breaking change and not keeping this broken behavior.